<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color: DimGrey;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="play">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px; font: 8pt &quot;Arial&quot;; font: 8pt &quot;Arial&quot;; font-weight: bold; color: white; transition: .2s linear; background: Gainsboro; hover box-shadow: 0 0 0 2px white, 0 0 0 4px #3C82F8;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>160</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px; font: 8pt &quot;Arial&quot;; font: 8pt &quot;Arial&quot;; font-weight: bold; color: white; transition: .2s linear; background: Gainsboro; hover box-shadow: 0 0 0 2px white, 0 0 0 4px #3C82F8;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>41</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">html{
  background:#e74c3c;
}

form{
  width:260px;
  height:260px;
  margin:0 auto;
}

input[type=&quot;range&quot;]{
  -webkit-appearance:none;
  width:160px;
  height:20px;
  margin:10px 50px;
  background: linear-gradient(to right, #9A2720 0%, #9A2720 100%);
  background-size:150px 10px;
  background-position:center;
  background-repeat:no-repeat;
  overflow:hidden;
  outline: none;
}

input[type=&quot;range&quot;]:first-of-type{
  margin-top:30px;
}

input[type=&quot;range&quot;]::-webkit-slider-thumb{
  -webkit-appearance:none;
  width:20px;
  height:20px;
  background:#F26B5E;
  position:relative;
  z-index:3;
  box-shadow:0 0 5px 0 rgba(0,0,0,0.3);
}

input[type=&quot;range&quot;]::-webkit-slider-thumb:after{
  content:&quot; &quot;;
  width:160px;
  height:10px;
  position:absolute;
  z-index:1;
  right:20px;
  top:5px;
  background: #ff5b32;
  background: linear-gradient(to right, #f088fc 1%, #AC6CFF 70%);
}

p{
  width:100%;
  text-align:center;
  color:white;
  font-family: 'HelveticaNeue-UltraLight', 'Helvetica Neue UltraLight', Roboto, Arial, Sans-serif;
  font-size:32px;
  margin-top:30px;
}
</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="push_button">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>250</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>538</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuplayer">
    <property name="title">
     <string>player</string>
    </property>
    <addaction name="actionopen_file"/>
   </widget>
   <addaction name="menuplayer"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionopen_file">
   <property name="text">
    <string>open file</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
